packages/types/src/json.ts

```diff
@@ -380,15 +380,6 @@ export interface OrganizationSuggestionJSON extends ClerkResourceJSON {
   updated_at: number;
 }
 
-export interface OrganizationSuggestionJSON extends ClerkResourceJSON {
```

Comment:
Note: This is an exact copy of the interface above it which ESLint didn't catch until now.